rfc/0054-sdk3-management-apis.md

```diff
@@ -1090,9 +1090,25 @@ public interface IAnalyticsIndexManager{
      void DisconnectLink(DisconnectLinkAnalyticsOptions options);
 
      map[string]map[string]int GetPendingMutations(GetPendingMutationsAnalyticsOptions options);
+     
+     void CreateLink(AnalyticsLink link, CreateLinkAnalyticsOptions options);
+     
+     void ReplaceLink(AnalyticsLink link, ReplaceLinkAnalyticsOptions options);
+     
+     void DropLinkFromScope(string linkName, string scopeName, DropLinkAnalyticsOptions options);
+     
+     void DropLinkFromDataverse(string linkName, string dataverseName, DropLinkAnalyticsOptions options);
+     
+     void GetLinkFromScope(string linkName, string scopeName, GetLinkAnalyticsOptions options);
```

Comment:
Should these GetLinkFrom* methods return `AnalyticsLink` instead of `void`? And GetAllLinks return `List<AnalyticsLink>`?

Reply:
Yes, will update tomorrow.